src/libs/elektra/kdb.c

```diff
@@ -1597,52 +1593,51 @@ static bool runGetPhase (KeySet * backends, Key * parentKey, uint16_t phase)
  * Retrieve Keys from the Key database in an atomic and universal way.
  *
  * @pre The @p handle must be a valid KDB handle as returned from kdbOpen().
- * @pre The @p returned KeySet must be a valid KeySet, e.g. constructed
- *     with ksNew().
- * @pre The @p parentKey Key must be a valid Key, e.g. constructed with
- *     keyNew().
+ * @pre The KeySet @p returned must be a valid KeySet, i.e., constructed with ksNew().
+ * @pre The KeySet @p returned must contain keys only from the `spec:/`, `dir:/`, `user:/`, `system:/`, `default:/` or `proc:/` namespaces.
+ * @pre The @p parentKey Key must be a valid Key, i.e., constructed with keyNew().
```

Comment:
analog changes also with Key:

```suggestion
 * @pre The Key @p parentKey must be a valid Key, i.e., constructed with keyNew().
```